source/mainnet/net/snippets/delegation-faq.rst

```diff
@@ -1,164 +1,163 @@
 .. _delegation-faq:
 
-Delegation and baking FAQ
-=========================
+Delegation and validation FAQ
+=============================
 
```

Comment:
Somewhere in all this there should be new question(s) about the variable commissions, e.g., how can I know what pool has what commission, how can I be notified that my pool is changing commission, can I change pool if the commission in my pool changes.